dev/com.ibm.etools.maven.liberty.integration/src/com/ibm/etools/maven/liberty/integration/manager/internal/MavenProjectInspector.java

```diff
@@ -123,12 +125,14 @@ public boolean isSupportedProject(IProgressMonitor monitor) {
     }
 
     public boolean isMavenProject() {
-        IFile pom = project.getFile(LibertyMavenConstants.POM_FILE_NAME);
-        if (pom == null || !pom.exists()) {
-            Trace.trace(Trace.INFO, "The project " + project.getName() + " is not a Maven Project");
-            return false;
+        try {
+            IProjectDescription description = project.getDescription();
```

Comment:
Just curious why it is doing this instead of just calling Project.hasNature?

Reply:
Hi @eharris369 , it was in the Gradle Project Inspector.  It should have used this API originally. I've updated both now.